perses/rjmc/topology_proposal.py

```diff
@@ -3586,3 +4325,93 @@ def _make_skewed_atom_map(self, topology):
                 atom_map[bond[0].index] = bond[0].index
                 atom_map[bond[1].index] = bond[1].index
         return atom_map
+
+class NetworkXMolecule(object):
```

Comment:
this generates a networkx representation of molecules for chirality and ring querying in `FFAllAngleGeometryEngine`